### ISSUE TITLE
suggestion: explain toggles / profile / menu entries in a wiki

### ISSUE THREAD

**Issue by tusch001** [scoute-dich/browser#741]
Although I've been using FOSS Browser for a long time I would like more info about some toggles. I couldn't find it on github. The FAQs.md is there but still doesn't answer all questions. E.g.
- what is the DOM toggle for? (I thought without Javascript there is no access to DOM for a website)
- what is blocked by fingerprinting?
or profiles:
- are utm_... parameters in urls blocked by the browser in the protected profile?

___

I think that new users with no background in programming are quite lost with this many toggles and profile options. Esp. since there is not much space for labels. 

A structured help - menu / submenu behind the settings info icon or a link to a github wiki would be helpful. Even if it is only available in English.

___

I would be willing to expand the explanations for some additional browser functions (with my limited English). But I lack the knowledge how the browser handles the details so I would need help at some point.  

___

Are you are interested in this?  

If so:
maybe you can suggest a format (html code as dt / link or ul / li /a lists in files or a wiki page with several md-files?).
I could start with creating a general structure. We could go from there and discuss what should be included.

**Comment by scoute-dich** [comment]
Good idea. I would prefer markdown. There is a wikipage at: https://github.com/scoute-dich/browser/wiki

_I started this project once for my own to learn developing. I shared my app on XDA, then on F-Droid and so on. The project grew over the years.  FOSS Browser is translated in over 15 languages ... Most of them would not be possible with the help of this small community._

_**I don't have the time to create and maintain a wiki. But if you'll help, it could be possible. Feel free to start!**_

**Comment by woheller69** [comment]
Fingerprint protection:

-Block WebRTC requests which can reveal local IP address
-Prevent canvas fingerprinting by randomizing
-Prevent WebGL fingerprinting by randomizing
-Prevent AudioContext fingerprinting by randomizing
-Prevent Font fingerprinting by randomizing
-Do not allow ping on http only pages
-Spoof screen resolution and color depth
-Set random values for device memory, hardwareConcurrency
-Remove battery, network connection, keyboard, media devices info
-Prevent sendBeacon

**Comment by scoute-dich** [comment]
I will start work in Christmas holidays. Since then, you all can collect information here or somewhere else where I can access it. Can you also edit the wiki? I think contributors can ...
